### ISSUE TITLE
Unable to open project ( "Can not connect the storage." )

### ISSUE THREAD

**Issue by nathanvv** [microsoft/BotFramework-Composer#3552]
## Describe the bug

Not able to use the open feature of bot frame work composer

![image](https://user-images.githubusercontent.com/11080029/85875760-5826b600-b79a-11ea-9d8f-3680a38d7f31.png)

## Version

![image](https://user-images.githubusercontent.com/11080029/85875800-67a5ff00-b79a-11ea-985c-b54c552db96d.png)

## Browser

Bot framework composer desktop application
- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x ] Windows
- [ ] Ubuntu

## To Reproduce

1. Open the Botframework composer desktop application
2. Click on 'open'
3. See error "Can not connect the storage." ( The first screenshot has the error )

## Expected behavior

To provide a browser view to open an downloaded composer project

## Screenshots

![image](https://user-images.githubusercontent.com/11080029/85876075-cbc8c300-b79a-11ea-90d4-b2cc414b25e1.png)

## Additional context

developer tool error : 

![image](https://user-images.githubusercontent.com/11080029/85876377-3b3eb280-b79b-11ea-8dfa-f2efec0bdb30.png)

**Comment by nathanvv** [comment]
@liweitian : Closing the app does not help. The desktop application is an electron app that still runs the web app in the background right?. I am able to do developers tools in the app 

![image](https://user-images.githubusercontent.com/11080029/86154386-a0124980-bac8-11ea-83cd-7399ca5e9393.png)

Notice on the error in console . Its just holding to that path, how do I clear that path, tried clearing the cookies and cache. Seems like the path "c:/repos/***********" is just getting stored somewhere . Can you point me to the settings file that holds that information, so that I can try clearing from there?

**Comment by liweitian** [comment]
@nathanvv Sorry for late reply. The Path "c:/repos/***********" is surely stored somewhere. But you cannot see it in the an electron app. And I still need some more info on this bug then I can help to debug it. Could you try on the link here https://github.com/microsoft/BotFramework-Composer#build-composer-locally, I need to know what happened on the backend. You can paste the error on the terminal here.

**Comment by nathanvv** [comment]
@liweitian: Tried doing that with the latest master and I ended up unsuccessful in replicating the issue. However, found a workaround to clear it. Apparent the path to the project I was opening via the app had space, but when downloading the folder got ASCII converted and had a %20 in the folder name ( notice screenshot )
![image](https://user-images.githubusercontent.com/11080029/87864903-cd1d8380-c933-11ea-9639-48f24b0482d1.png)
When trying to open via composer it was looking for the name 'Intern Playground' ( not Intern%20Playground) and it was complaining that the folder was missing. The composer was converting the ASCII to space and it was holding the path somewhere. So just to work around I renamed the folder to have space and now I am finally able to open projects again 
![image](https://user-images.githubusercontent.com/11080029/87864938-2e455700-c934-11ea-981d-2bd65156b5b4.png)
But feel like the last opened path should be easier to clear. Even uninstalling the application didn't end up clearing that. 
Thanks for the help and response though. Greatly appreciated.
